#errorlar bilan ishlash

# print(x)#bunday xato beradi chunki x degan uzgaruvchi yuq

# a=4
# n=0
# print(a/n) #bu xato qaytaradi

# ls=[1,2,3]
# print(ls[10])#10chi indexsni chiqar deyapman xato beradi


# try:#bu xatoni oldini olish programani tuxtatmaydi
#                       ls=[1,2,3]
#                       print(ls[10])#bu xato chunki 10idex yuq lsda
# except:
#                       print('xato tepada ls ichida 10 index yuq')#tepadagi xatoni tushintirish



